pkgs/tools/filesystems/apfs-fuse/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, fuse3, bzip2, zlib, attr, cmake }:
+
+stdenv.mkDerivation rec {
+  name = "apfs-fuse-${version}";
```

Comment:
Please use `pname = "apfs-fuse";` as `name = "${name}-${version}";` is already implied.